docs/vercel.json

```
@@ -1,8 +1,84 @@
 {
     "redirects": [
         {
-          "source": "/version-comparison",
-          "destination": "https://www.flagsmith.com/pricing"
+        "source": "/advanced-use/staged-feature-rollouts",
+        "destination": "https://docs.flagsmith.com/best-practices/when-to-use-flags#staged-rollouts"
+        },
+        {
+        "source": "/api/bulk-insert-identities-update",
+        "destination": "https://docs.flagsmith.com/edge-api/bulk-insert-identities-update"
+        },
+        {
+        "source": "/clients/3rd-party",
+        "destination": "https://docs.flagsmith.com/integrating-with-flagsmith/sdks/"
+        },
+        {
+        "source": "/clients/server-s",
+        "destination": "https://docs.flagsmith.com/integrating-with-flagsmith/sdks/server-side"
+        },
+        {
+        "source": "/deployment/configuration/sizing-and-scaling",
+        "destination": "https://docs.flagsmith.com/deployment-self-hosting/scaling-and-performance/sizing-and-scaling"
+        },
+        {
+        "source": "/deployment/hosting/kubernetes",
+        "destination": "https://docs.flagsmith.com/deployment-self-hosting/hosting-guides/kubernetes-openshift"
+        },
+        {
+        "source": "/enterprise/overview/",
+        "destination": "https://docs.flagsmith.com/deployment-self-hosting/enterprise-edition"
+        },
+        {
+        "source": "/guides-and-examples/efficient-api-usage",
+        "destination": "https://docs.flagsmith.com/best-practices/efficient-api-usage"
+        },
+        {
+        "source": "/guides-and-examples/mobile-app-versioning",
+        "destination": "https://docs.flagsmith.com/best-practices/mobile-app-versioning"
+        },
+        {
+        "source": "/integrations/apm/appdynamics",
+        "destination": "https://docs.flagsmith.com/deployment-self-hosting/scaling-and-performance/monitoring#appdynamics"
+        },
+        {
+        "source": "/integrations/datadog/",
+        "destination": "https://docs.flagsmith.com/third-party-integrations/observability-and-monitoring/datadog"
+        },
+        {
+        "source": "/integrations/importers/launchdarkly",
+        "destination": "https://docs.flagsmith.com/administration-and-security/data-management/import-from-launchdarkly"
+        },
+        {
+        "source": "/integrations/overview/",
+        "destination": "https://docs.flagsmith.com/integrating-with-flagsmith/integration-overview"
+        },
+        {
+        "source": "/integrations/project-management/github",
+        "destination": "https://docs.flagsmith.com/third-party-integrations/#project-management--communication"
```

Not sure what the logic was behind this redirect but it should be as follows.  

```suggestion
        "destination": "https://docs.flagsmith.com/third-party-integrations/project-management/github"
```